camayoc/constants.py

```diff
@@ -15,6 +15,8 @@
     'jboss.eap.packages',
     'jboss.eap.processes',
     'jboss.eap.running-paths',
+    'jboss.eap.locate-jboss-modules-jar',
```

Comment:
Please sort alphabetically the entries to follow how other fact lists are organized in code. I remember you using vim, so it will be a matter of `vi):sort` while the cursor is in the the first list item.